sdk/spring/azure-spring-cloud-starter-eventhubs/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.microsoft.azure</groupId>
+  <artifactId>spring-cloud-starter-azure-eventhubs</artifactId>
+  <version>1.2.8-beta.1</version> <!-- {x-version-update;com.microsoft.azure:spring-cloud-starter-azure-eventhubs;current} -->
+
+  <name>Azure Spring Cloud Starter Event Hubs</name>
```

Comment:
Should all of these starters have "Starter" in their names? There's an inconsistency in the <name> strings with the libraries in this PR.

Reply:
Thank you for your careful check.
We plan to update name after artifact name updated.
Here is new created issue: https://github.com/Azure/azure-sdk-for-java/issues/13953